app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @recipes.each_with_index do |recipe,index|  %>
+<%= link_to recipe_path(recipe.id) do %>
+<div class='recipe-container'>
+<section class='recipe-info'>
+   <h3 class='recipe-1'>Recipe <%= index+1 %></h3>
+   <a class='recipe-btn'><%= button_to "Remove", recipe_path(recipe), method: :delete, data: { turbolinks: false } %></a>
+</section>   
+   <p class='recipe-description'><%= recipe.description %></p>
+</div>  
+ <% end %>  
+<% end %>
+
```

Comment:
![image](https://github.com/ismayito/Rails-recipe-app/assets/59397159/c904812a-cdbb-4dc7-9be4-40c938449a00)

![image](https://github.com/ismayito/Rails-recipe-app/assets/59397159/2e38abd1-2d7e-408b-9d72-59b12b12b9eb)

---

- Your recipe should have this wireframe with the appropriate conten. Currently, it is empty. please, fix this.